src/snowflake/snowpark/session.py

```diff
@@ -2429,6 +2430,11 @@ def write_pandas(
             table_type: The table type of table to be created. The supported values are: ``temp``, ``temporary``,
                 and ``transient``. An empty string means to create a permanent table. Learn more about table types
                 `here <https://docs.snowflake.com/en/user-guide/tables-temp-transient.html>`_.
+            use_logical_type: Boolean that specifies whether to use Parquet logical types when reading the parquet files
+                for the uploaded pandas dataframe. With this file format option, Snowflake can interpret Parquet logical
+                types during data loading. To enable Parquet logical types, set use_logical_type as True. Set to None to
+                use Snowflakes default. For more information, see:
+                `file format options: <https://docs.snowflake.com/en/sql-reference/sql/create-file-format#type-parquet>`_.
```

Comment:
is it only supported starting from a certain version of connector?
should we log warning if use_logical_type not in signature.parameters but still specified by user in this case?

Reply:
we do that already

Reply:
yep, this is what I was asking, thanks for pointing